client/spec/helpers/pages.js

```diff
@@ -216,3 +217,30 @@ function IngestDashboard() {
         return dashboard.element(by.css('.ingested-count'));
     };
 }
+
+function MasterDesks() {
```

Comment:
This should be defined in the desks module; we should put e2e tests in the corresponding modules.